common/position.go

```diff
@@ -102,6 +102,6 @@ func (p CdcPosition) ToSDKPosition() opencdc.Position {
 	return v
 }
 
-// TableKeys is a collection of associations between table names and primary key
-// names. Useful for retrieving primary key values from rows.
-type TableKeys map[string]string
+// TableSortColumns is a map of associations between table names and sort column
+// names. Useful for retrieving sort values from rows.
+type TableSortColumns map[string]string
```

Comment:
Sorry @hariso, what do you mean? We don't really need this separate type if that's what you are asking, but makes the code a bit more readable, don't you think? After all, I have to obtain the table keys at the source, and pass that data down to the iterators. If I did pass a raw map[string]string, well, you start to rely on struct property naming to know what is it.

I do like having a distinct type for that, even if there's a single map[string]string. Isn't it like `conduit-commons/config.Config` type?

Reply:
> If I did pass a raw map[string]string, well, you start to rely on struct property naming to know what is it.

I think that's fine, because we can always give the fields and parameters proper names (comments too if needed), and that should do the work. If we say we need a type to know what a variable is about, we would basically be saying that we need to use a custom type for almost everything.

> I do like having a distinct type for that, even if there's a single map[string]string. Isn't it like `conduit-commons/config.Config` type?

The `Config` type is a bit different, since we have some custom methods for it.

Reply:
Fair enough, let's remove it then.